IFTB.bs

```diff
@@ -0,0 +1,535 @@
+     zero-offset-change entries if needed.
+
+Note that the offset array itself is not required to be stored at any particular location in the table.
+
+### Table Ordering ### {#iftb-table-ordering}
+
+No two tables in an IFTB font should share a tag name.
+
+In a CFF or CFF2 based IFTB font that table must be last. In a glyf/loca based IFTB font the last
+tables must be glyf and loca in that order (loca following glyf is a requirement inherited from WOFF2).
+If the font contains a gvar table it must come immediately before the glyf table.
+
+There are no other ordering requirements. However, it is highly recommended that the IFTB and cmap
+tables come early in the list and before any other lengthy table.
+
+IFTB Font Formats {#iftb-font-formats}
+--------------------------------------
+
+The specification allows for two initial font formats:
+
+1.   The uncompressed format is a slight variant of the OpenType "sfnt" format, but with a version tag
+     of <code>IFTB</code> instead of <code>0x00010000</code> or <code>OTTO</code>
+2.   The compressed format is a WOFF2 encoding of the uncompressed format with all tables in the same
+     order.  The sfnt "flavor" in the WOFF2 header must accordingly also be <code>IFTB</code>
+
+It is expected that the client will almost always download a WOFF2-encoded file, which it will then
+unpack into the uncompressed format and modify as needed.
+
+Before loading a copy of the uncompressed font into the browser, or making it available to some other
+renderer, the version should be changed to <code>0x00010000</code> if the font has a glyf table or
+<code>OTTO</code> if it has a CFF or CFF2 table.  However, if the file is cached it should be stored
+with the <code>IFTB</code> version.  (The differing version is to help prevent an IFTB-encoded font
+from being used accidentally as a standard font, which will generally result in blank spaces where some
+glyphs should be rendered.)
+
+IFTB Table {#iftb-table}
+------------------------
+
+The IFTB table contains information needed to determine what chunks are needed for a given font subset
+description, and then to retrieve and integrate those chunks. It is similar in layout to analogous
+OpenType tables, and is accordingly described in terms of the [OpenType Specification Data
+Types](https://learn.microsoft.com/en-us/typography/opentype/spec/otff#data-types). As with OpenType
+all fields use Big Endian byte orderng.
+
+### IFTB Table Header ### {#iftb-table-header}
+
+```
+uint16     majorVersion   - set to 1
+uint16     minorVersion   - set to 0
+uint32     reserved       - not used, set to 0
+uint32     id[4]          - ID uniquely identifying this IFTB font
+uint16     flags          - not yet used
+uint32     chunkCount     - number of bins in this IFTB font
+uint32     glyphCount     - must match field in maxp table
+Offset32   CFFCharStringsOffset  - 0 if glyf-based
+Offset32   gidMapOffset
+Offset32   chunkOffsetListOffset
+Offset32   featureMapOffset
```

Comment:
Here and elsewhere Offset fields are used be sure to mention what those offsets are relative too.